author	￼Khem Raj <raj.khem@gmail.com>
committer	￼Stephen Hemminger <stephen@networkplumber.org>	
commit	ae717baf15fb4d30749ada3948d9445892bac239
tree	4d4632ba5e9af8cb29e0cbc14190c3138945facf
parent	a2325adf0fbadf8b3184aa22208871ad97551012 

tc: include stdint.h explicitly for UINT16_MAX
Fixes
| tc_core.c:190:29: error: 'UINT16_MAX' undeclared (first use in this function); did you mean '__INT16_MAX__'?
|    if ((sz >> s->size_log) > UINT16_MAX) {
|                              ^~~~~~~~~~

Signed-off-by: Khem Raj <raj.khem@gmail.com>

diff --git a/tc/tc_core.c b/tc/tc_core.c
index 7bbe0d7..821b741 100644
--- a/tc/tc_core.c
+++ b/tc/tc_core.c
@@ -12,6 +12,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdint.h>
 #include <unistd.h>
 #include <syslog.h>
 #include <fcntl.h>
